msteams-platform/bots/build-a-bot.md

```diff
@@ -0,0 +1,58 @@
+| 3. | Bot registration service | - Azure AD <br> - Developer Portal |
+| 4. | Advanced bot capabilities | - Call and media bot <br> - Access data using MS Graph <br> - SSO <br> - Bot on Mobile |
+
+## Build a bot
+
+Let's start by building a conversational bot. You can use Teams Toolkit, Bot Framework SDK, or Teams AI library. After you build a conventional Teams bot app, you can add the AI layer to your bot.
+
+### Build a conventional Teams bot app
+
+| # | I want to build ... | Here's a sample |
+| --- | --- | --- |
+| 1. | Conversational bot using Bot Framework SDK | - [Bot app with Node.js](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation/nodejs) <br> - [Bot app with C#](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation/csharp) <br> - [Bot app with Python](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation/python) |
+| 2. | Conversational bot using Teams Toolkit | [Create Teams conversation bot](../sbs-teams-conversation-bot.yml) |
+
+### Build an AI-powered Teams bot app
+
+You can build an AI bot or elevate your existing conventional bot to be powered by AI.
+
+| # | I want to build ... | Here's how... |
+| --- | --- | --- |
+| 1. | AI bot with Teams AI library and Teams Toolkit | [Build a custom engine agent](../Teams-AI-library-tutorial.yml) |
+| 2. | Elevate your conventional bot to AI | [Elevate your conventional bot to use AI](how-to/teams-conversational-ai/how-conversation-ai-get-started.md#elevate-your-conventional-bot-to-use-ai) |
+| 3. | Migrate bot to use Teams AI library | [Migrate to the Teams AI library](https://github.com/microsoft/teams-ai/tree/main/getting-started/migration) |
```

Comment:
Word choices-- I feel like the distinction between 2 ("Elevate") and 3 ("Migrate") is somewhat vague. My suggestion is to emphasize the framework names, like, "Elevate your conventional bot to use AI with Bot framework adapter" and "Migrate bot built with Bot framework to Teams AI Library".